kong/tools/utils.lua

```diff
@@ -48,6 +48,7 @@ const char *ERR_reason_error_string(unsigned long e);
 
 int open(const char * filename, int flags, int mode);
 size_t read(int fd, void *buf, size_t count);
+int write(int fd, const void * ptr, int numbytes);
```

Comment:
Slightly pedantic, but eventually we should probably agree upon asterisk placement for pointers in the function prototypes we put in `ffi.cdef`.

Reply:
Ah yep. This was a copypasta. I agree, I don't like the whitespace between the defer op and the symbol. Fixed.